src/ff/Makefile

```diff
@@ -1,4 +1,5 @@
-FFLAGS        = 
+FC            = gfortran
```

Comment:
The Makefile changes here are the https://github.com/conda-forge/ff-feedstock/blob/7ad7719ed0a64b8f05914b3c9acb3779d72e8f57/recipe/update-makefile.patch in https://github.com/conda-forge/ff-feedstock.